imports/collections/schemas/shops.js

```diff
@@ -204,6 +204,22 @@ export const ParcelSize = new SimpleSchema({
 
 registerSchema("ParcelSize", ParcelSize);
 
+/**
+ * @name ShopLogoUrls
+ * @memberof Schemas
+ * @type {SimpleSchema}
+ * @property {String} primaryShopLogoUrl optional
+ * @property {String} styles optional
+ */
+export const ShopLogoUrls = new SimpleSchema({
+  primaryShopLogoUrl: {
+    type: String,
+    optional: true
+  }
+});
+
+registerSchema("ShopLogoUrls", ShopLogoUrls);
```

Comment:
I don't see this being used anywhere. Should the field just be added to shop schema?

Reply:
Yes.